pkg/sdk/logging/model/syslogng/config/render/helpers.go

```diff
@@ -134,6 +134,18 @@ func Literal[T LiteralTypes](v T) Renderer {
 	}
 }
 
+func StringList(stringList []string) Renderer {
```

Comment:
a very simple test for this would be good I think, even if it's already tested through the redis output implicitly

Reply:
Good idea, thanks! Added it in c5afd53.